File: src/trapizza.py
from math import pi

d = int(input())
a = int(input())
b = int(input())
h = int(input())
m = pi * d * d / 4
t = (a + b) * h / 2
if m == t:
    print("Jafn storar!")
elif m > t:
    print("Mahjong!")
else:
    print("Trapizza!")

File: src/trapizza.lua
d, a, b, h = io.read("*n", "*n", "*n", "*n")
m = math.pi * d * d / 4
t = (a + b) * h / 2
if m == t then
    print('Jafn storar!')
elseif m > t then
    print('Mahjong!')
else
    print('Trapizza!')
end
